client/src/api/rebrick.js

```diff
@@ -0,0 +1,12 @@
+export async function getSets(query) {
+  const url = `https://rebrickable.com/api/v3/lego/sets/?search=${query}`;
```

Comment:
You should call this API from your own server to make it possible to hide the authorization key from the client and have more control over the transferred data.
Thus, you should create a new endpoint in your API which forwards the request to the rebrickable API and returns the data back to the client. Maybe convert it if it makes sense (the properties are not in camelCase -> you could map it on server side).

Reply:
Done!